site/src/pages/UserSettingsPage/SecurityPage/SecurityPage.test.tsx

```diff
@@ -76,24 +76,6 @@ test("update password with incorrect old password", async () => {
 	expect(API.updateUserPassword).toBeCalledWith(user.id, newSecurityFormValues);
 });
 
-test("update password with invalid password", async () => {
-	jest.spyOn(API, "updateUserPassword").mockRejectedValueOnce(
-		mockApiError({
-			message: "Invalid password.",
-			validations: [{ detail: "Invalid password.", field: "password" }],
-		}),
-	);
-
-	const { user } = await renderPage();
-	fillAndSubmitSecurityForm();
-
-	const errorMessage = await screen.findAllByText("Invalid password.");
-	expect(errorMessage).toBeDefined();
-	expect(errorMessage).toHaveLength(2);
-	expect(API.updateUserPassword).toBeCalledTimes(1);
-	expect(API.updateUserPassword).toBeCalledWith(user.id, newSecurityFormValues);
-});
-
```

Comment:
I think we instead modify this test to spy on the validate user password endpoint.